services/web-app/components/mdx-page/mdx-page.js

```diff
@@ -92,14 +80,28 @@ const MdxPage = ({
   warnings,
   children,
   pageHeaderType,
-  seoTitle
+  metaTitle
 }) => {
   const { setPrimaryNavData } = useContext(LayoutContext)
   const router = useRouter()
   const areTabsPresent = tabs && tabs.length > 0
+  let defaultMetaTitle = useMetaTitle([], areTabsPresent)
+
   const pathSegments = router.asPath.split('/').filter(Boolean)
   pathSegments.pop()
   const baseSegment = pathSegments.join('/')
+
+  const tabsData = tabs.map((tab) => {
+    if (tab?.name && tab?.path) {
+      return tab
+    }
+    const tabSlug = slugify(tab, { strict: true, lower: true })
+    return {
+      name: tab,
+      path: baseSegment ? `/${baseSegment}/${tabSlug}` : `/${tabSlug}`
+    }
+  })
```

Comment:
Tabs can be not supplied: https://github.com/carbon-design-system/carbon-platform/blob/main/services/web-app/pages/libraries/%5Bhost%5D/%5Borg%5D/%5Brepo%5D/%5Blibrary%5D/%5Bref%5D/assets/%5Basset%5D/%5Btab%5D.js#L86
```suggestion
  const tabsData = tabs?.map((tab) => {
    if (tab?.name && tab?.path) {
      return tab
    }
    const tabSlug = slugify(tab, { strict: true, lower: true })
    return {
      name: tab,
      path: baseSegment ? `/${baseSegment}/${tabSlug}` : `/${tabSlug}`
    }
  })
```